assisted_deployment.sh

```diff
@@ -7,18 +7,181 @@ source common.sh
 source assisted_deployment_utils.sh
 
 
+ASSISTED_INSTALL_TEST_INFRA=${ASSISTED_INSTALL_TEST_INFRA:-}
+ASSISTED_INSTALLER_NAMESPACE="${ASSISTED_INSTALLER_NAMESPACE:-assisted-installer}"
+ASSISTED_OPERATOR_INDEX="${ASSISTED_OPERATOR_INDEX:-quay.io/ocpmetal/assisted-service-index:latest}"
+
+
+function deploy_local_storage() {
+  oc adm new-project openshift-local-storage || true
+
+  oc annotate project openshift-local-storage openshift.io/node-selector=''
+
+  cat <<EOF | oc apply -f -
+apiVersion: operators.coreos.com/v1alpha2
+kind: OperatorGroup
+metadata:
+  name: local-operator-group
+  namespace: openshift-local-storage
+spec:
+  targetNamespaces:
+    - openshift-local-storage
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: local-storage-operator
+  namespace: openshift-local-storage
+spec:
+  channel: "4.7"
+  installPlanApproval: Automatic
+  name: local-storage-operator
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+EOF
+
+  wait_for_crd "localvolumes.local.storage.openshift.io"
+
+  echo "Creating local volume and storage class..."
+  cat <<EOCR | oc apply -f -
+apiVersion: local.storage.openshift.io/v1
+kind: LocalVolume
+metadata:
+  name: assisted-service
+  namespace: openshift-local-storage
+spec:
+  logLevel: Normal
+  managementState: Managed
+  storageClassDevices:
+    - devicePaths:
+        - /dev/sdb
+        - /dev/sdc
+      storageClassName: assisted-service
+      volumeMode: Filesystem
+EOCR
+}
+
+
+function deploy_hive() {
```

Comment:
does this obsolete hive_install.sh?

Reply:
Likely, and prob also worth removing the test-infra based install below.

I wanted to leave any removals/depreciations for a follow up PR as I'm not sure what may be using this code.